Userland/Services/WindowServer/MultiScaleBitmaps.cpp

```diff
@@ -25,6 +25,17 @@ const Gfx::Bitmap& MultiScaleBitmaps::bitmap(int scale_factor) const
     return it->value;
 }
 
+Gfx::Bitmap const* MultiScaleBitmaps::find_bitmap(int scale_factor) const
+{
+    auto it = m_bitmaps.find(scale_factor);
+    return it != m_bitmaps.end() ? it->value.ptr() : nullptr;
+}
+
+RefPtr<MultiScaleBitmaps> MultiScaleBitmaps::create_empty()
```

Comment:
Couldn't this return a `NonnullRefPtr`?

Reply:
No, if there isn't at least one valid bitmap then it needs to be returning a nullptr, otherwise the `bitmap()` function can't guarantee to return a valid reference to a bitmap.